src/views/SsoSettings.vue

```diff
@@ -395,6 +400,11 @@ export default defineComponent({
   },
 
   computed: {
+    documentListAPIPath(): string {
+      const apiPrefix = `${this.apiRoot}/${this.apiVersion}`
+      return `${apiPrefix}/api/v3/reblaze/tools/sso/`
```

Comment:
`/api/v3` is already included in the root and version

```suggestion
      return `${apiPrefix}/reblaze/tools/sso/`
```

Reply:
Done